chapters/001-Routing/1.2-smol-internet/README.md

```diff
@@ -338,8 +338,8 @@ Sweet! It looks like that's working! But, what does this request look like on `b
 
 Let's use `tcpdump` to investigate how `boudi` processes these packets. To do that, we'll need a couple more terminal windows open, one to run the `ping` from `tara` and one for each interface on `boudi`.
 
-Window 1: `tara` will run `ping 10.1.1.3 -c 2`
-Window 2: `boudi` will run `tcpdump -ni eth0`
+Window 1: `tara` will run `ping 10.1.1.3 -c 2`  
+Window 2: `boudi` will run `tcpdump -ni eth0`  
```

Comment:
Huh. Markdown linting added this spacing? That's weird. 🤷

Reply:
No, the spacing added is mine - it's so they'll newline in the md formatting output.

"Rachel's List:
Thing1
Thing2"

becomes

"Rachel's List:Thing1Thing2"

So for _ as space, if you do:
"Rachel's List:__
Thing1__
Thing2__"

it will output as desired, each one on their own line.